Gordon360/AuthorizationFilters/StateYourBusiness.cs

```diff
@@ -646,6 +711,16 @@ private bool canDelete(string resource)
                     return false;
                 case Resource.ADMIN:
                     return false;
+                case Resource.HOUSING:
+                    {
+                        // Only the superadmins and the housing admins can delete members from the whitelist
+                        var housingService = new HousingService(new UnitOfWork());
+                        if (user_position == Position.SUPERADMIN || (housingService.CheckIfHousingAdmin(user_id)))
```

Comment:
This should only be `if (user_position == Position.SUPERADMIN)` since the act of modifying the whitelist in any way is supposed to be restricted to only the superadmins.
This resource should also be changed to `Resource.HOUSING_ADMIN`